lib/jobs/ScheduleBackupJob.js

```diff
@@ -139,15 +139,15 @@ class ScheduleBackupJob extends BaseJob {
     return Promise.try(() => {
       const jobData = _.cloneDeep(jobOptions);
       jobData.attempt = jobData.attempt + 1;
-      if (jobData.attempt > config.scheduler.jobs.scheduled_backup.max_attempts) {
+      if (jobData.attempt > _.get(jobData, 'max_attempts', config.scheduler.jobs.scheduled_backup.max_attempts)) {
         logger.error(`Scheduled backup for instance  ${jobData.instance_id} has exceeded max configured attempts : ${config.scheduler.jobs.scheduled_backup.max_attempts} - ${jobData.attempt}}. Initial attempt was done @: ${jobData.firstAttemptAt}.`);
```

Comment:
can you change the log statement also to print the value correctly for  for max_attempts and reschedule_delay ?

Reply:
Done!